### DISCUSSION TITLE
Failed to load resource: the server responded with a status of 404 (Not Found) livewire.js

### DISCUSSION THREAD

**Discussion by Gopal-Bandyopadhyay** [livewire/livewire#5929] [7. Bugs (archived)] [answered]
Livewire V3 above error is showing "Failed to load resource: the server responded with a status of 404 (Not Found) Livewire.js" when testing the counter-example given in the documentation.
Kindly help to how to solve the issue, in V2, i have solved the issue using assets URL link

**Comment by joshhanley** [comment]
@Gopal-Bandyopadhyay can you provide some more information about what OS you are using, how you have PHP installed, etc. so we can understand better what environment this issue is occurring in. Also can you share the code for the fix you applied in V2? Thanks!

**Comment by Gopal-Bandyopadhyay** [comment]
I use Php 8.1 and Laravel 10 versions on a Windows 10 PC with a Wamp server
and getting errors.
in V2 I have applied Asset_URL in the Livewire config file for linking the
Livewire js file.

**Reply by Gopal-Bandyopadhyay** [reply]
Problem Resolved after adding project folder in routing.

Livewire::setUpdateRoute(function ($handle) {
   return Route::post('Autogen_V3_Livewire_3/public/livewire/update', $handle);
});

Livewire::setScriptRoute(function ($handle) {
   return Route::get('Autogen_V3_Livewire_3/public/livewire/livewire.js', $handle);
});

**Comment by Gopal-Bandyopadhyay** [comment, marked as answer]
Problem Resolved after adding project folder in routing.

Livewire::setUpdateRoute(function ($handle) {
   return Route::post('Autogen_V3_Livewire_3/public/livewire/update', $handle);
});

Livewire::setScriptRoute(function ($handle) {
   return Route::get('Autogen_V3_Livewire_3/public/livewire/livewire.js', $handle);
});

**Reply by royallife88** [reply]
Not working for me giving below error using with Filament Laravel
`Unable to prepare route [subdir/livewire/update] for serialization. Another route has already been assigned name [livewire.update]`
Any help?

**Reply by timinigi** [reply]
But why asset url variable was removed? I have different url only in local setup, adding these callbacks to Provider (therefore to actual code, not just .env variable) seems cluttering and unnecessary.
